### DISCUSSION TITLE
How to use suspense with urql?

### DISCUSSION THREAD

**Discussion by quinn** [urql-graphql/urql#1651] [Q&A] [answered]
Hi, i've followed a lot of github issues, PRs, npm packages, etc that all seem to indicate that suspense support is now part of the urql library, but I haven't found a good explanation of how to use it. Is this documented anywhere?

**Comment by JoviDeCroock** [comment, marked as answer]
The usage on the client is pretty in line with the `lazy` concept. You initialize the Client with suspense: true and it will throw up to Suspense boundaries. In ssr you'll need to use a tool like react-ssr-prepass as Suspense on the server isn't supported in React yet.

[SSR docs](https://formidable.com/open-source/urql/docs/advanced/server-side-rendering/#using-react-ssr-prepass)

**Reply by quinn** [reply]
"suspense: true" was the secret sauce i was looking for. Couldn't find that documented anywhere. thank you!

**Reply by JoviDeCroock** [reply]
All right, that's good to know will make a note to be clearer about that in the docs 🙌 thanks for the feedback
